EnoEngine/FlagSubmissionEndpoint.cs

```diff
@@ -116,7 +116,7 @@ async Task ReadPipeAsync(PipeReader reader, ChannelWriter<Task<FlagSubmissionRes
                     // Process the line.
                     if (teamId is long _teamId)
                     {
-                        var flag = Flag.Parse(line, EnoDatabaseUtils.FLAG_SIGNING_KEY, Logger);
+                        var flag = Flag.Parse(line, Encoding.ASCII.GetBytes(config.FlagSigningKey), Logger);

```

Comment:
I think we should have a second config class which contains the key as bytes, and pass that around via DI, but that can go in another PR at a later time